snap/info.go

```diff
@@ -997,3 +997,28 @@ func DropNick(nick string) string {
 	}
 	return nick
 }
+
+// StoreName splits the instance name and returns the store name of the snap.
+func StoreName(name string) string {
```

Comment:
Nitpick: please name the argument `instanceName` (also below)

Reply:
renamed